RSSReader/View/FeedViewController.m

```diff
@@ -0,0 +1,87 @@
+//
+//  FeedViewController.m
+//  RSSReader
+//
+//  Created by Владислав Станкевич on 17.11.20.
+//
+
+#import "FeedViewController.h"
+#import "FeedCell.h"
+
+@interface FeedViewController ()
+
+@end
+
+@implementation FeedViewController
+
+- (void)viewDidLoad {
+    self.navigationController.navigationBar.hidden = YES;
+    [super viewDidLoad];
+    [self setupPresenter];
+    [self setupTableView];
+    [self setupConstraints];
+}
+
+- (void)setupPresenter {
+    NSMutableArray *feedItemArray = [[NSMutableArray alloc] init];
+    self.feedItemArray = feedItemArray;
+    [feedItemArray release];
+    FeedPresenter *presenter = [[FeedPresenter alloc] initWithArray:self.feedItemArray];
```

Comment:
What is the purpose of passing an empty array into the initialization call of FeedPresenter??